framework/application/modules/Session/Session.php

```diff
@@ -364,32 +395,6 @@ public function setPostLogin($post_login=REDIRECT_AFTER_LOGIN)
 	/*** private methods ***/
 
 	/**
-	 * setMessage
```

Comment:
I've been using PHPStorm "refactor" which organizes the code but PHPStorm has an option to keep setters and getters at the top but it does not recognize our setters because of all the logic in them. I have an idea how I can still get it to keep them at the top.